src/index.html

```diff
@@ -16,16 +16,16 @@
         <div class="row">
           <div class="col text-left">
             <ul>
-              <li><a href="#">USD <i class="fas fa-caret-down"></i></a></li>
-              <li><a href="#">English <i class="fas fa-caret-down"></i></a></li>
-              <li><a href="#">Help <i class="fas fa-caret-down"></i></a></li>
+              <li class="header-first-li"><a href="#">USD <i class="fas fa-caret-down"></i></a></li>
+              <li class="header-first-li"><a href="#">English <i class="fas fa-caret-down"></i></a></li>
+              <li class="header-first-li"><a href="#">Help <i class="fas fa-caret-down"></i></a></li>
             </ul>
           </div>
           <div class="col text-right">
             <ul>
-              <li><a href="#"><i class="fas fa-user"></i> Login</a></li>
-              <li><a href="#"><i class="fas fa-lock"></i> Register</a></li>
-              <li><a href="#"><i class="fas fa-bars"></i></a></li>
+              <li class="header-contact-li"><a href="#"><i class="fas fa-user"></i> <span class="header-links">Login</span></a></li>
```

Comment:
what does `li` mean in that class name?

please don't use tag names in CSS classes, you'll have them separated better and you'll not run into situation when you'd like to use `header-contact-li` for other (eg. `div` element)